client/src/components/OpenTicket/TheReply/style.css

```diff
@@ -0,0 +1,76 @@
+.text_reply{
+    margin-left: -15px;
+}
+.reply{
+    margin-left: 295px;
+}
+.content_reply{
+    width: 631px;
+    height: 300px;
+    margin-top: 10px;
```

Comment:
collapse the margins into one line.
margin: 10px 0 0 300px;
also, using fixed pixels for margins might lead to responsiveness issues. Keep that in mind.

Reply:
i used only margin-top not margin-top and other margin so here not found collapse margin 
but , i now modify in many files in code css => collapse margin